Rocky.class.nut

```diff
@@ -8,6 +8,16 @@ class Rocky {
 
     static PARSE_ERROR = "Error parsing body of request";
```

Comment:
Can you please move all these static variables into constants? Please use a prefix`ROCKY_` for each of them?

Reply:
Constants take much less space than statics (are inlined at preprocessing time) and are faster to access.

Reply:
Sure thing.